code/modules/clothing/outfits/standard.dm

```diff
@@ -178,6 +178,16 @@
 	mask = /obj/item/clothing/mask/cigarette/cigar/havana
 	l_pocket = /obj/item/lighter
 
+/datum/outfit/mr_runtime
+	name = "Mr. Runtime"
```

Comment:
I would be more clear about this, it is not out of the question that we will add more of these.

1. Rename this to something explicit, like `stacks_in_hands`
2. Add a doc comment explaining precisely what this is for, and the issue it exists to fix. Otherwise, someone will remove it because they think it's dead code.